device/arista/x86_64-arista_7280r4_32qf_32df/Arista-7280R4-32QF-32DF-64O/buffers_defaults_t2.j2

```diff
@@ -0,0 +1,56 @@
+{%- set default_cable = '300m' %}
+
+{%- set ports2cable = {
+        'torrouter_server'       : '300m',
+        'leafrouter_torrouter'   : '300m',
+        'spinerouter_leafrouter' : '2000m',
+        'upperspinerouter_spinerouter' : '30m',
+        'upperspinerouter_lowerspinerouter' : '30m',
+        'regionalhub_upperspinerouter': '120000m',
+        'aznghub_upperspinerouter'    : '120000m',
+        'regionalhub_spinerouter': '120000m',
+        'aznghub_spinerouter'    : '120000m'
+        }
+-%}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,128,8) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{%- endif %}
+    {%- endfor %}
+    {%- for port_idx in range(128,256,4) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{%- endif %}
+    {%- endfor %}
+    {%- for port_idx in range(256,384,8) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "12884901888",
```

Comment:
@arista-nwolfe  As per my understanding, total memory should be 32GB per chip but above number seems to be driven from 16GB. Is that a mistake ? Similar concern for xoff param.